src/event_handler.js

```diff
@@ -49,6 +49,10 @@ export function initWorker() {
             case "getInt":
                 sendAmountOfVariableToWorkerAsInput(message);
                 break;
+            case "createObject":
+                gameController.createObject(message.details);
+            case "destroyObject":
+                gameController.destroyObject(message.details);
```

Comment:
all cases except default should have "break" so multiple cases aren't executed

Reply:
true